src/Shared/BrowserTesting/src/Directory.Build.props

```diff
@@ -0,0 +1,8 @@
+<Project>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\..\..\, Directory.Build.props))\Directory.Build.props" />
```

Comment:
What's with this? Why is it jumping over dirs?

Reply:
Other projects on the area do it like this, there are no Directory.Build.props files in between.

Reply:
`GetDirectoryNameOfFileAbove(...)` itself will find the Directory.Build.props file in the repo root. Should never use more than one :..\` because it gets in the way of moving files around. The single `..\` is needed to avoid finding the same Directory.Build.props file.

Reply:
I wasn't aware of that, I'll update it